src/Tribe/Service_Providers/Context.php

```diff
@@ -44,32 +45,33 @@ public function register() {
 	 * @return array The modified context locations.
 	 */
 	public function filter_context_locations( array $locations = [] ) {
-		$locations = array_merge(
+		$view_query_var = View::query_var();
+		$locations      = array_merge(
 			$locations,
 			[
-				'event_display'        => [
+				'event_display'      => [
 					'read'  => [
 						Tribe__Context::WP_MATCHED_QUERY => [ 'eventDisplay' ],
 						Tribe__Context::WP_PARSED        => [ 'eventDisplay' ],
-						Tribe__Context::REQUEST_VAR      => [ 'view', 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
+						Tribe__Context::REQUEST_VAR      => [ $view_query_var, 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
 						Tribe__Context::QUERY_VAR        => 'eventDisplay',
 						Tribe__Context::TRIBE_OPTION     => 'viewOption',
 					],
 					'write' => [
-						Tribe__Context::REQUEST_VAR => [ 'view', 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
+						Tribe__Context::REQUEST_VAR => [ $view_query_var, 'tribe_view', 'tribe_event_display', 'eventDisplay' ],
 						Tribe__Context::QUERY_VAR   => 'eventDisplay',
 					],
 				],
-				'view'                 => [
+				$view_query_var      => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 8 space(s) between "$view_query_var" and double arrow, but found 6.

<sub>hash: 1dd2e9c8333f264748287785e438b414</sub>